src/Catty/ViewController/Stage/StagePresenterSideMenuView.swift

```diff
@@ -281,6 +290,11 @@ enum SideMenuButtonType {
         button.frame.size.width = CGFloat(StagePresenterSideMenuView.buttonInitialWidthAndHeight)
         button.frame.size.height = CGFloat(StagePresenterSideMenuView.buttonInitialWidthAndHeight)
 
+        if (imageName == "stage_dialog_button_back" || imageName == "stage_dialog_button_continue" || imageName == "stage_dialog_button_restart") && UIDevice.current.orientation.isLandscape {
```

Comment:
Please use `self.landscape` instead of `UIDevice.current.orientation.isLandscape` to check whether landscape mode is enabled.